cirq-google/cirq_google/ops/internal_gate.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict
+from cirq import ops, value
+
+
+@value.value_equality
+class InternalGate(ops.Gate):
+    """InternalGate is a placeholder gate for internal gates.
+
+    InternalGate holds the information required to instantiate
+    a gate of type `self.gate_name` with the arguments for the gate
+    constructor stored in `self.gate_args`.
+    """
+
+    def __init__(self, gate_name: str, gate_module: str = 'pyle', num_qubits: int = 1, **kwargs):
```

Comment:
gate_module should probably default to 'pyle.cirqtools.pyle_gates'

Reply:
done